manifests/agent/configure.pp

```diff
@@ -31,7 +31,7 @@
     Optional[Array[String]] $tags          = $logdna::params::conf_tags,
     Optional[String] $hostname             = $logdna::params::conf_hostname,
     Optional[Array[String]] $exclude       = $logdna::params::conf_exclude,
-    Optional[Array[String]] $exclude_regex = $logdna::params::conf_exclude_regex
+    Optional[String] $exclude_regex        = $logdna::params::conf_exclude_regex
```

Comment:
This should still be an array. The template file will convert it back down to a single line string.

Reply:
I returned it back to being an array after merging but in `logdna-agent`, we take only one string value for the regex for exclusion.

Thanks again for this Pull Request!